public/app/listeners/executePythonScript.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+const { spawn } = require('child_process');
+const logger = require('../logger');
+
+const executePythonScript = (event, args) => {
+  const { scriptId } = args;
+  logger.debug(`executing script with scriptId ${scriptId}`);
+  logger.debug('expecting a file named copy_of_data.json to exist');
+  logger.debug(__dirname);
```

Comment:
Is this log necessary?

Reply:
Nope, just kept it there for debugging. I will remove it.